app/components/molecules/section-name/section-name.component.tsx

```diff
@@ -0,0 +1,49 @@
+import cult from './img/cult.svg';
+import menu from './img/menu.svg';
+import temple from './img/temple.svg';
+import {
+  Label,
+  LeadingIcon,
+  StyledSectionName,
+  Border,
+} from './section-name.styles';
+
+export const Menu = () => (
+  <StyledSectionName>
+    <LeadingIcon src={menu} />
+    <Label>
+      EL MENÚ
+      <Border />
+    </Label>
+  </StyledSectionName>
+);
+
+export const Cult = () => (
+  <StyledSectionName>
+    <LeadingIcon src={cult} />
+    <Label>
+      EL CULTO
+      <Border />
+    </Label>
+  </StyledSectionName>
+);
+
+export const Temple = () => (
+  <StyledSectionName>
+    <LeadingIcon src={temple} />
+    <Label>
+      EL TEMPLO
+      <Border />
+    </Label>
+  </StyledSectionName>
+);
+
+export const Community = () => (
+  <StyledSectionName>
```

Comment:
I still dont get why if this is pretty much the same style as the others and the only thing that changes are:

1. LeadingIcon src
2. Chidren text

Reply:
I feel you could simplify all this file like this or similar. 😄 

```typescript
const labels = {
  'cult': 'EL CULTO',
   ...
}

const icon = {
  'cult': cult,
   ...
}

interface Props {
 type: 'cult' | 'etc' | '...';
}

export const Section = ({ type }) => (
  <StyledSectionName>
    <LeadingIcon src={icon[type]} />
    <Label>
       labels[type]
      <Border />
    </Label>
  </StyledSectionName>
);
```